cli/lsp/completions.rs

```diff
@@ -3,7 +3,6 @@
 use super::client::Client;
 use super::config::ConfigSnapshot;
 use super::config::WorkspaceSettings;
-use super::documents::file_like_to_file_specifier;
```

Comment:
@nayeemrmn instead of doing the specifier mapping for jupyter notebooks between the language server and TSC I believe we can just do it between the client and server as done in this PR and fixes the issue. Is there any issue with doing it this way? It simplifies the code.

Reply:
`file:///path/to/notebook.ipynb#cell1` isn't a real way of reading that module content from the perspective of the file fetcher or other systems, so in theory it could break somewhere. Those systems are typically overrided for open documents so I'm not surprised if our tests and regular usage seem to work fine. But I think the server's doc storage should maintain an accurate view of these specifiers.

Reply:
On main, it seems to map them to file specifiers anyway though? For simplicitly it seems to work ok because the LSP doesn't read open documents from the disk ever. Maybe we should only take on the complexity of exposing the notebook scheme once we have a scenario that shows it's necessary?